ha/core/system_health/entity_health.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
Move this file to ha/core/system_health/model/ directory. This is data model which will be stored.

Reply:
ok

Reply:
done